x/ccv/provider/types/keys.go

```diff
@@ -7,6 +7,8 @@ import (
 	"time"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	utils "github.com/cosmos/interchain-security/x/ccv/utils"
```

Comment:
Nit: we have a lot of things that we refer to as utils
```suggestion
	ccvutils "github.com/cosmos/interchain-security/x/ccv/utils"
```